mlflow/environment_variables.py

```diff
@@ -34,6 +38,24 @@ def __repr__(self):
         return repr(self.name)
 
 
+class _BooleanEnvironmentVariable(_EnvironmentVariable):
+    """
+    Represents a boolean environment variable.
+    """
+
+    def get(self):
+        if not self.is_defined:
+            return self.default
+
+        val = os.getenv(self.name)
+        lowercased = val.lower()
+        if lowercased not in ["true", "false", "1", "0"]:
+            raise ValueError(
+                f"{self.name} value must be one of ['true', 'false', '1', '0'], but got {val}"
+            )
+        return lowercased == "true"
```

Comment:
`return lowercased == "true" or lowercased == "1"`

Reply:
thanks for the catch, fixed